go/kbfs/libkbfs/kbfs_ops.go

```diff
@@ -267,12 +267,62 @@ func (fs *KBFSOpsStandard) GetFavorites(ctx context.Context) (
 
 // GetFavoritesAll implements the KBFSOps interface for
 // KBFSOpsStandard.
-func (fs *KBFSOpsStandard) GetFavoritesAll(ctx context.Context) (keybase1.
-	FavoritesResult, error) {
+func (fs *KBFSOpsStandard) GetFavoritesAll(ctx context.Context) (
+	keybase1.FavoritesResult, error) {
 	timeTrackerDone := fs.longOperationDebugDumper.Begin(ctx)
 	defer timeTrackerDone()
 
-	return fs.favs.GetAll(ctx)
+	favs, err := fs.favs.GetAll(ctx)
+	if err != nil {
+		return keybase1.FavoritesResult{}, err
+	}
+
+	// Add the conflict status for any folders in a conflict state to
+	// the favorite struct.
+	journalManager, err := GetJournalManager(fs.config)
+	if err != nil {
+		// Journaling not enabled.
+		return favs, nil
+	}
+	conflicts, err := journalManager.GetJournalsInConflict(ctx)
+	if err != nil {
+		return keybase1.FavoritesResult{}, err
+	}
+
+	if len(conflicts) == 0 {
+		return favs, nil
+	}
+	conflictMap := make(map[ConflictJournalRecord]tlf.ID, len(conflicts))
+	for _, c := range conflicts {
+		conflictMap[ConflictJournalRecord{Name: c.Name, Type: c.Type}] = c.ID
+	}
+
+	found := 0
+	for i, f := range favs.FavoriteFolders {
```

Comment:
Should we consider also doing this for new folders? It would be confusing to end up in a conflict state due to CLI shenanigans and then not be able to see it in the GUI.

Reply:
I don't think there is a way for new folders to be edited without appearing in the favorites list.  By "CLI shenanigans" do you mean writing to the TLF via a terminal?  Any access in the terminal automatically puts a TLF on your favorites list.

Reply:
I meant `keybase fs` and/or FUSE, I guess.
>  Any access in the terminal automatically puts a TLF on your favorites list.
In that case, then I agree it's not necessary.

Reply:
Yeah, `keybase fs` works the same way, TLFs get auto-added to favorites.